src/config/dynamic-theme-fixes.config

```
@@ -1656,6 +1656,20 @@ IGNORE INLINE STYLE
 
 ================================
 
+biorxiv.org
+
+INVERT
+.blood_logo
+.logo-img
+#czilogo
+
+CSS
+body {
+    background: none !important;
```

```suggestion
    background-image: none !important;
```
Let's be a little more specific to target.